### DISCUSSION TITLE
Feature Request: Add an icon option to attribute row type in entities card

### DISCUSSION THREAD

**Discussion by tomlut** [home-assistant/frontend#9470] [Dashboard]
The schema for attributes as rows in entities cards does not currently support an icon option see erroneously opened issue: https://github.com/home-assistant/frontend/issues/9461

I humbly request that it be added.

An older card I made before this schema test was introduced actually works (displays the correct icons), however it does generate a lot of errors when the card is opened for editing.

I can help by updating the documentation if you like.

**Comment by mbo18** [comment]
I vote for! I have a card that was working fine with `icon:` but I needed to update it and now I'm unable to add `icon: `back...

**Comment by spacegaier** [comment]
I created a PR to tackle that: https://github.com/home-assistant/frontend/pull/9963
